sdk/tables/azure-data-tables/azure/data/tables/_base_client.py

```diff
@@ -269,8 +281,8 @@ def _batch_send(self, table_name: str, *reqs: HttpRequest, **kwargs) -> List[Map
         # Pop it here, so requests doesn't feel bad about additional kwarg
         policies = [StorageHeadersPolicy()]
 
-        changeset = HttpRequest("POST", None)  # type: ignore
-        changeset.set_multipart_mixed(*reqs, policies=policies, boundary=f"changeset_{uuid4()}")  # type: ignore
+        changeset = HttpRequest("POST", "")
+        changeset.set_multipart_mixed(*reqs, policies=policies, boundary=f"changeset_{uuid4()}")
         request = self._client._client.post(  # pylint: disable=protected-access
```

Comment:
This code should be updated. The generated code uses azure.core.rest.HttpRequest, while we're still creating azure.core.transport.HttpRequest here.

I think this is a bug in disguise here, that we're not consistent. 

It may be adressed in a different PR, but then we should have an issue for that.

To showcase that tables is using the new system, just look at imports done by the generated code:
https://github.com/Azure/azure-sdk-for-python/blob/96a11ae924ac8842d4c00c7e6419c15788c1f376/sdk/tables/azure-data-tables/azure/data/tables/_generated/operations/_operations.py#L22

cc @annatisch

Reply:
Logged an issue

Reply:
I've started a separate PR for this here:
https://github.com/Azure/azure-sdk-for-python/pull/32418